src/views/settings/Msg/MsgTemplate/index.vue

```diff
@@ -128,7 +133,7 @@ export default {
       this.$axios.get('/api/v1/notifications/templates/').then(data => {
         if (data.length > 0) {
           this.templates = data
-          this.fieldsMeta.EMAIL_TEMPLATE_NAME.el.options = data.map(item => ({
+          this.fieldsMeta.template_name.el.options = data.map(item => ({
             label: item.subject,
             value: item.template_name
           }))
```

Comment:
The provided code snippet was cut off. Please provide the full code to proceed with your inquiries about checking differences, reviewing issues, or generating optimizations.